tasks/query-cmr/app/stac/cmr-catalog.ts

```diff
@@ -102,6 +102,7 @@ export default class CmrStacCatalog extends StacCatalog {
           properties: {
             start_datetime: granule.time_start,
             end_datetime: granule.time_end,
+            datetime: null,
```

Comment:
I agree with your suggestion of using the start time - so `granule.time_start` instead of `null` here I think.

Reply:
Should note that it seems strange to have datetime, start_datetime, and end_datetime, but probably better to just make sure the validation succeeds.

Reply:
Just had another thought - we should probably be consistent with CMR STAC catalogs - do they have these 3 datetime properties?

Reply:
Looking at the code, yes they do: https://github.com/nasa/cmr-stac/blob/master/src/domains/items.ts#L174